document/CoreFormat.mdk

```diff
@@ -205,11 +205,11 @@ The information in the `runInfo` object makes it possible to precisely repeat a
 and to verify that the issues reported in the log file were generated by an appropriate invocation of the tool.
 ~
 
-### `commandLineArguments` property { #runInfo-commandLineArguments }
+### `parameterization` property { #runInfo-parameterization }
 
-A `runInfo` object shall contain a property named `commandLineArguments` whose value is a string
-containing the command line arguments with which the tool was invoked.
-This string shall not include the file name or path to the executable itself.
+A `runInfo` object may contain a property named `parameterization` whose value is a string
+containing the parameterization with which the tool was invoked.
```

Comment:
Consider changing "the parameterization with which" to "the run-time parameters with which".